bentoml/_internal/server/metrics/prometheus.py

```diff
@@ -83,41 +103,59 @@ def mark_process_dead(self) -> None:
             self.prometheus_client.multiprocess.mark_process_dead(self._pid)
 
     def start_http_server(self, port: int, addr: str = "") -> None:
-        self.prometheus_client.start_http_server(
+        self.prometheus_client.start_http_server(  # type: ignore
             port=port,
             addr=addr,
             registry=self.registry,
         )
 
-    def generate_latest(self):
+    def generate_latest(self) -> bytes:
         if self.multiproc:
             registry = self.prometheus_client.CollectorRegistry()
             self.prometheus_client.multiprocess.MultiProcessCollector(registry)
-            return self.prometheus_client.generate_latest(registry)
+            return t.cast(bytes, self.prometheus_client.generate_latest(registry))
         else:
             return self.prometheus_client.generate_latest()
 
     def get_metrics_report(
```

Comment:
nits: shall we maintain the analytics code outside of the `server/` directory here? They are irrelevant. Putting them here looks like we are excluding those paths from the `/metrics` results as well.

Reply:
imo `get_metrics_report` should filter out those paths from the report's perspective, and I'm ok with leaving those here.